src/craftdef.cpp

```diff
@@ -236,7 +236,7 @@ static void craftDecrementOrReplaceInput(CraftInput &input,
 			}
 		}
 		// No replacement was found, simply decrement count to zero
```

Comment:
Btw, does it always decrement the count **to zero**? From how I see it, it only decrements **by one**. The comment should perhaps be fixed.